OloEngine/src/OloEngine/Networking/Replication/SpatialGrid.cpp

```
@@ -0,0 +1,172 @@
+
+    u32 SpatialGrid::GetEntityCount() const
+    {
+        return static_cast<u32>(m_EntityCells.size());
+    }
+
+    f32 SpatialGrid::GetCellSize() const
+    {
+        return m_CellSize;
+    }
+
+    void SpatialGrid::SetCellSize(f32 cellSize)
```

Clang (14.0.6)

[warning] method 'SetCellSize' can be made static

(readability-convert-member-functions-to-static)